docker/elasticsearch-setup/ml-model-index-config.json

```diff
@@ -0,0 +1,225 @@
+{
+  "settings": {
+    "index": {
+      "analysis": {
+        "filter": {
+          "autocomplete_filter": {
+            "type": "edge_ngram",
+            "min_gram": "3",
+            "max_gram": "50"
+          },
+          "custom_delimiter": {
+            "split_on_numerics": "false",
+            "split_on_case_change": "false",
+            "type": "word_delimiter",
+            "preserve_original": "true",
+            "catenate_words": "false"
+          }
+        },
+        "char_filter": {
+          "ml_model_pattern": {
```

Comment:
The char_filter here gets used on every ml_model_pattern (used in the name field). What it does is replace all the dots with slashes, why it does this or what the significance of it is, I'm not exactly sure, I copied it from Dataset. It doesn't seem to be used from what I can tell. It would make sense for example if the slash pattern got utilized with this character filter as it would allow for tokenizing of a name like "aws_us_east_1.modelName" (which would get pattern replace to aws_us_east_1/modelName and then be two separate tokens from splitting on the slash) or something like that. 

Do you have any insight from the LinkedIn side why dots are undesirable in the name field?

Reply:
There are couple of issues I can see here with the mappings and settings. It will be great if you can confirm the same
1. `name` field is using `model_pattern` analyzer but that doesn't seem to be defined anywhere? I do see `ml_model_pattern` analyzer being defined but not `model_pattern` analyzer.
2. `ml_model_pattern` analyzer makes use of `ml_model_pattern` tokenizer but where are we using the char_filter `ml_model_pattern`?

Reply:
To answer your question on why we replaced dots with slashes in datasets is to support browse feature. We chose this uniform standard since some datasets name were of form `foo.bar` while others of form `/foo/bar`. You don't need this in your settings/mappings if you don't intend to support browse.
This also means that we should improve our documentation :) Thanks for the question.